installer.sh

```diff
@@ -520,3 +487,13 @@ if [[ "$TARBALL" -eq "1" ]] ; then
     fi
     ./make_agent_bundle_tarball.sh $TAR_BUNDLE_FLAGS
 fi
+
+echo "=================================================================================="
+echo $'\t\t\t\tInstallation complete!'
+echo "=================================================================================="
+if [[ -n "$(command -v dnf)" ]] || [[ -n "$(command -v yum)" ]]; then
+    if [[ "$TPM_VERSION" -eq "2" ]] && [[ "$TPM_SOCKET" -eq "1" ]] ; then
+        echo "As you have selected to run an emulator, you will need to disable SELinux"
```

Comment:
Set to permissive instead?
If the emulator ever works in an SELinux enabled system, it's easier to set back to enforce than having to relabel from disabled state.

Reply:
That's a good point @dawud. thanks..will make the change.

Reply:
Are we sure this is needed?  I've run the emulator + abrmd in Fedora before without needing to disable SELinux, but you need to install the policy: https://github.com/tpm2-software/tpm2-abrmd/blob/master/INSTALL.md#selinux

Reply:
It always fails for me and I am pretty sure i found a denial.

Reply:
```
285. 09/20/2019 19:03:20 tpm2-abrmd system_u:system_r:tabrmd_t:s0 0 udp_socket create system_u:system_r:tabrmd_t:s0 denied 1009
286. 09/20/2019 19:03:20 tpm2-abrmd system_u:system_r:tabrmd_t:s0 0 udp_socket create system_u:system_r:tabrmd_t:s0 denied 1010
287. 09/20/2019 19:03:20 tpm2-abrmd system_u:system_r:tabrmd_t:s0 0 tcp_socket create system_u:system_r:tabrmd_t:s0 denied 1011
288. 09/20/2019 19:03:20 tpm2-abrmd system_u:system_r:tabrmd_t:s0 0 tcp_socket create system_u:system_r:tabrmd_t:s0 denied 1012
```

I will try and get the label applied. 

I need to dig more as well, as javier had a `tpm2-abrdmd-selinux` package as a depency to the main abrmd package.

Reply:
@jetwhiz I just noticed we don't even build tpm2-abrmd:

https://github.com/keylime/keylime/blob/01785a25077fbb6b2c2220395a3fd99300583d43/installer.sh#L394-L419

Reply:
Yeah we don't install it by default with the installer, since I wasn't able to get a cross-platform, automated way to get it working.  The instructions are there to get it working in Ubuntu, and the README also shows how to install it.  It seems to be very finicky, though, especially on Fedora.

Reply:
Ack, makes sense. i remember having to do it differently with Fedora as well. It should be fine now, as its ubuntu specific.